orca-api/src/main/java/com/netflix/spinnaker/orca/api/SimpleStage.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.api;
+
+import com.google.common.annotations.Beta;
+
+/**
+ * Used to create a more simplistic stage. This should be used by plugin developers to create plugin
+ * stages
```

Comment:
`... more simplistic stage.` more than what? I'd just get rid of that sentence imo. These docs are a little more anemic than I'd prefer to see, but we can work with them as we go along and we figure more assumptions, etc.